### ISSUE TITLE
!@shhubham tag doesn’t works 

### ISSUE THREAD

**Issue by ShhubhamA** [bitclout/frontend#350]
Small bug -If there is a Exclamation mark before @ it doesn’t  tag  the user 
Example 

![429D8F57-3BC1-4F28-BA29-DEA5F9220EA1](https://user-images.githubusercontent.com/71114510/127752576-fbbc9d1d-a7a0-40c0-be43-cecced841463.jpeg)

**Comment by AdityaChaudhary0005** [comment]
yeah, infact pronunciation marks after tag don't work too. like `@ItsAditya,` `@ItsAditya!` , `@ItsAditya?` , `@ItsAditya^` etc
